ui/packages/ui/src/main/webapp/js/views/installer/InstallerMain.view.js

```diff
@@ -48,7 +48,7 @@ define([
         },
         changePage: function() {
             //close whatever view is open
-            var welcomeStep = 0, guestClaimsStep = 1, profileStep = 2, configStep = 3,  finishStep = 4;
+            var welcomeStep = 0, guestClaimsStep = 2, profileStep = 1, configStep = 3,  finishStep = 4;
```

Comment:
Is there any documentation that needs to be updated?

Reply:
I didn't find any documentation that specified the order which is the only thing that changed.

Reply:
Looking again I think I might need to change the `:order:` number in the headers so they get added in the same order they appear in the installer.